products/mautic.md

```diff
@@ -139,3 +158,7 @@ releases:
 > [Mautic](https://www.mautic.org/), is an open source marketing automation software.
 
 Mautic only actively supports the latest version with new features, bug fixes, and security updates.
+
```

Comment:
```
{: .note }
```
Making this as a note will make it looks much prettier

Reply:
We did not do it for other products, will keep it like this as I don't want to put too much emphasis on commercial support.